scripts/repos.txt

```diff
@@ -38,3 +38,4 @@ fxa-bugzilla-mirror
 grunt-l10n-lint
 fxa-activity-metrics
 mozlog
+mozilla-services/fxa-testing-e2e
```

Comment:
The code in `ghlib.js` already has support for parsing this "owner/repo" syntax, I just needed to make it look up the right owner in a couple of places, for ease of use.